src/endpoints/getMatchesStats.ts

```diff
@@ -12,17 +12,19 @@ export const getMatchesStats = (config: HLTVConfig) => async ({
   startDate,
   endDate,
   matchType,
-  maps = []
+  maps = [],
+  rankingFilter
 }: {
   startDate?: string
   endDate?: string
   matchType?: MatchType
   maps?: Map[]
+  rankingFilter?: string
```

Comment:
Can you set the type to `RankingFilter` instead?

Reply:
Yes, this seemed a little more straight forward but I agree an enum is better.